### ISSUE TITLE
Multiple Database support failing due to config error

### ISSUE THREAD

**Issue by valhuber** [ApiLogicServer/ApiLogicServer-src#24]
Adding a database to a project can cause errors like this when accessing database via swagger:

```
Generic Error: (sqlite3.OperationalError) no such table
```

**Comment by valhuber** [comment]
Investigation indicates that you can fix this by updating `config.py` to designate the proper url, eg.

```
    SQLALCHEMY_DATABASE_URI_TODO = 'sqlite:///../database/Todo_db.sqlite'    # repair uri like this
    app_logger.info(f'config.py - SQLALCHEMY_DATABASE_URI_TODO: {SQLALCHEMY_DATABASE_URI_TODO}\n')
```

**Comment by valhuber** [comment]
Fixed in 9.05.08
